### ISSUE TITLE
Attach to process - Unexpected token N in JSON at position 50177 on Windows

### ISSUE THREAD

**Issue by Valzul** [microsoft/vscode-cpptools#9821]
Just updated my VSCode to 1.70.2 on Windows 10, and it seems like I cannot attach to process anymore when wanting to debug C++ ( I got C/C++ extension ) application.

my launch.json is the following : 

```cpp
{
  "configurations": [
    {
      "name": "Attach to process",
      "type": "cppvsdbg",
      "request": "attach",
      "processId" :"${command:pickProcess}"
    }
  ],
  "version": "2.0.0"
}
```

Here is the result : 
![vscode_unexpected_token](https://user-images.githubusercontent.com/72873887/187717345-33c34d52-f78a-4509-9d2d-fc4485601b6a.JPG)

It seems to work when the request is "launch" to a specific program.

Could this be a problem on my hand ?

Thanks for your help

**Comment by Colengms** [comment]
Hi @Valzul .  Could you clarify which version of the C/C++ Extension you are using?

I tried creating a simple program, building it with cl.exe, and attaching to it using the `launch.json` content you provided.  I was able to attach and hit a breakpoint.

Does the issue repro for you with a simple example, or only a particular project?  If the issue always repro's for you, there would seem to be something specific to your environment that we may need to uncover to investigate further.  I don't believe we've (yet?) received reports about this functionality being broken for other users.

**Comment by Valzul** [comment]
Here is my version of C/C++ extension : 

![vscode_cpp_extension_version](https://user-images.githubusercontent.com/72873887/187853863-b1eeb458-ef85-4fc5-91b9-6202c5a9a323.JPG)

I tried to create an empty folder with a single .cpp file and asked the creation of the debugging option settings file for C++
and launched it right away but it fired the error too.

Today on rebooting my computer, the problem is gone ...

It was the first time it happened to me yesterday. I successfuly used it the day before when I was still using the previous version of VSCode, I don't know what I did wrong.
